bin/public/installESpower.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+'''
+Install DHCP Server
+
+This dhcp server is mainly used to get KVM guest installation to work with
+kickstarts.
+
+http://www.yolinux.com/TUTORIALS/DHCP-Server.html
+http://www.linuxhomenetworking.com/wiki/index.php/Quick_HOWTO_:_Ch08_:_Configuring_the_DHCP_Server
+http://www.howtoforge.com/dhcp_server_linux_debian_sarge
+
+'''
+
+__author__ = "mattias.hemmingsson@fareoffice.com"
+__copyright__ = "Copyright 2014, The System Console project"
+__maintainer__ = "Mattias Hemmingsson"
+__email__ = "syco@cybercow.se"
+__credits__ = ["Daniel Lindh"]
+__license__ = "???"
+__version__ = "1.0.0"
+__status__ = "Production"
+
+import general
+from general import x
+from scopen import scOpen
+import app
+import version
+import os
+import iptables
+
+# The version of this module, used to prevent the same script version to be
+# executed more then once on the same host.
+SCRIPT_VERSION = 1
+CONF_SOURCE='/opt/syco/usr/syco-private/var/'
```

Comment:
dint now to what to change it to.
Added so it checks if there is a file in syco-private and if not uses /var instead ?